Dockerfile.dev

```diff
@@ -0,0 +1,33 @@
+ARG FEDORA_VERSION
+FROM fedora:${FEDORA_VERSION}
+
+WORKDIR /app
+
+# Package installation steps are separated so they
+# can be cached individually on Dockerfile changes
+
+RUN dnf upgrade --refresh -y
+
+# Kerberos packages
+RUN dnf install -y krb5-workstation krb5-devel krb5-libs
+
+# fedmsg (Federated Message Bus)
```

Comment:
You can remove this, the fedmsg should be removed from the-new-hotness in favor of fedora-messaging